src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagService.cs

```diff
@@ -47,17 +48,32 @@ public class AddExtendedQueryTagService : IAddExtendedQueryTagService
             _maxAllowedCount = extendedQueryTagConfiguration.Value.MaxAllowedCount;
         }
 
-        public async Task<AddExtendedQueryTagResponse> AddExtendedQueryTagsAsync(IEnumerable<AddExtendedQueryTagEntry> extendedQueryTags, CancellationToken cancellationToken = default)
+        public async Task<AddExtendedQueryTagResponse> AddExtendedQueryTagsAsync(
+            IEnumerable<AddExtendedQueryTagEntry> extendedQueryTags,
+            CancellationToken cancellationToken = default)
         {
             _extendedQueryTagEntryValidator.ValidateExtendedQueryTags(extendedQueryTags);
             var normalized = extendedQueryTags
                 .Select(item => item.Normalize())
                 .ToList();
 
             // TODO: Handle tags that have already been added
+            // Add the extended query tags to the DB
             IExtendedQueryTagStore extendedQueryTagStore = await _extendedQueryTagStoreFactory.GetInstanceAsync(cancellationToken);
-            IReadOnlyList<int> keys = await extendedQueryTagStore.AddExtendedQueryTagsAsync(normalized, _maxAllowedCount, ready: false, cancellationToken: cancellationToken);
-            string operationId = await _client.StartQueryTagIndexingAsync(keys, cancellationToken);
+            IReadOnlyCollection<int> expected = await extendedQueryTagStore.AddExtendedQueryTagsAsync(
```

Comment:
would it be better to call this statuses?

Reply:
These aren't status, like `ExtendedQueryTagStatus`. Instead these are the echoed tag keys for the new entries